src/fastFood.css

```diff
@@ -0,0 +1,36 @@
+*{
+  box-sizing: border-box;
+}
+#app{
+  margin: 200px;
```

Comment:
pixels are not a best choice if we are speaking about responsiveness, read about rems, ems.